sdk/iotcentral/Azure.ResourceManager.IotCentral/Directory.Build.props

```diff
@@ -1,4 +1,7 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsClientLibrary>true</IsClientLibrary>
```

Comment:
I don't think we need this, actually we use `IsMgmtLibrary` for all track 2 mgmt. plane sdk.